user_docs/en/userGuide.md

```diff
@@ -1806,6 +1806,30 @@ This option applies to all synthesizers, not just the currently active synthesiz
 On by default, this option tells NVDA if the current voice's language can be trusted when processing symbols and characters.
 If you find that NVDA is reading punctuation in the wrong language for a particular synthesizer or voice, you may wish to turn this off to force NVDA to use its global language setting instead.
 
+##### Unicode normalization {#SpeechSettingsUnicodeNormalization}
+| . {.hideHeaderRow} |.|
+|---|---|
+|Options |Default (Disabled), Enabled, Disabled|
+|Default |Disabled|
+
+When this option is enabled, unicode normalization is performed on the text that is spoken by NVDA.
+This is beneficial when speaking characters that can be represented in several forms.
+NVDA uses the NFKC (Normalization Form Compatibility Composition) algorithm, which provides the following benefits, among others:
+
+1. The bold and italic versions of characters that are part of the unicode standard and are commonly used on social media are normalized to their most common compatible equivalent.
+For example, the latin letter "h" can also be presented as "𝐡" (bold), "ℎ" (itallic), etc. but will always be spoken as "h" when normalization is enabled.
+
+1. Normalization to composed characters.
+For example, the character "ü" (u with umlaut/diaeresis), a common character in languages like German and Turkish can be represented in two forms.
+  1. One stand alone unicode character (ú)
+  1. A decomposition into two characters (ú), namely the normal latin letter u and a diaeresis modifier
+
+  Unicode normalization ensures that only one form will be used throughout all speech output, which is the one character variant.
+
+1. Decomposition of some ligatures, Including "ĳ" (ligature ij) to their two letter form ("ij").
+
+1. Stable ordering of modifiers in composite characters, for example in ancient Hebrew.

```

Comment:
```suggestion
1. Stable ordering of modifiers in composite characters, for example in ancient Hebrew.
```

Could you clarify the impact of this on speech?

Reply:
The modifiers will be spoken in normalized order rather than in the order they are in the text. For speech, this could be a bit confusing, but that's why this is optional. It has no impact on speech with the hebrew Vocalizer voice, for example. I'm afraid there is no normalization form where we can disable this behavior. On the other hand, for braille this can be of great benefit, as it requires a braille table to account for the normalized form of the modifiers only.